.github/workflows/publish_to_pypi.yml

```diff
@@ -0,0 +1,64 @@
+# Publish archives to PyPI and TestPyPI using GitHub Actions
+name: Publish to PyPI
+
+# Only run for tagged releases and pushes to the development branch
+on:
+  release:
+    types:
+      - published
+  push:
+    branches:
+      - development
+  # Runs for pull requests should be disabled other than for testing purposes
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  publish-pypi:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+    if: github.repository == 'icesat2py/icepyx'
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        # fetch all history so that setuptools-scm works
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2.2.1
+      with:
+        python-version: 3.9
+
+    - name: Install dependencies
+      run: python -m pip install setuptools wheel
+
+    # This step is only necessary for testing purposes and for TestPyPI
+    - name: Fix up version string for TestPyPI
+      if: ${{ !startsWith(github.ref, 'refs/tags') }}
+      run: |
+        # Change setuptools-scm local_scheme to "no-local-version" so the
+        # local part of the version isn't included, making the version string
+        # compatible with PyPI.
+        sed --in-place "s/node-and-date/no-local-version/g" setup.py
```

Comment:
Just to explain this section a little bit, what `sed` does is to replace `"local_scheme": "node-and-date"` in setup.py with `"local_scheme": "no-local-version"` on **non-tagged** pushes. So:

- On regular pushes to 'development', "no-local-version" is used which means a version string like `0.3.2.dev12`
- On proper tagged/published releases, "node-and-date" is used which means a version string like `0.3.3`

References:
- https://github.com/pypa/setuptools_scm/tree/v5.0.1#version-number-construction